package/MDAnalysis/coordinates/XDR.py

```diff
@@ -19,6 +19,18 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
+"""XDR based trajectory files --- :mod:`MDAnalysis.coordinates.XDR`
+===============================================================
```

Comment:
docstring of MDAnalysis.lib.formats.libmdaxdr:2: WARNING: Title underline too short.